SDK/AppCenter/Microsoft.AppCenter.Windows.Shared/AppCenter.cs

```diff
@@ -263,7 +263,7 @@ private AppCenter()
 
         private bool InstanceEnabled => _applicationSettings.GetValue(EnabledKey, true);
 
-        private Task SetInstanceEnabledAsync(bool value)
+        private Task SetInstanceEnabled(bool value)
```

Comment:
It could be in future and also it returns a task and thus can be called with await.
If await works it should have async in the name? Also the PCL public is async even if ios uses Task.fromResult for now, but ideally all disk accesses should be async in the future.

@achocron thougts?

Reply:
@guperrot This change caused by https://github.com/Microsoft/AppCenter-SDK-DotNet/pull/566#discussion_r160217877 and addressed to avoid misreading with locks

Reply:
It's in a confusing state right now since it returns a task but doesn't do async work i guess? I don't think that just returning a `Task` should make it `async` but if it is going to be `async` in the future maybe we should make it that way now to avoid conflicts in the future

Reply:
then I would consider building the task from result outside the method